.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature request
+    url: https://github.com/openpgpjs/openpgpjs/discussions/categories/ideas
+    about: Suggest an idea for this project
+  - name: Question
+    url: https://github.com/openpgpjs/openpgpjs/discussions
```

Probably this should be
```suggestion
    url: https://github.com/openpgpjs/openpgpjs/discussions/categories/q-a
```
?

---

I think it depends a bit, for instance I put this under "General" since it didn't seem library-specific enough for a Q&A...? https://github.com/openpgpjs/openpgpjs/discussions/1379